### PR TITLE
fix(demo): add all paging demos

### PR DESCRIPTION
**Pull request by kriswinbush**
## Description

Add demos for paging component
https://github.com/Teradata/covalent/issues/1630
### What's included?

- dynamic paging demo
- page links demo
- input page demo

#### Test Steps

- [ ] `npm run serve`
- [ ] select components
- [ ] click paging component

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

##### Screenshots or link to StackBlitz/Plunker
paging-demos

### CODE REVIEW

File: src/app/content/components/component-demos/paging/demos/paging-demo-input/paging-demo-input.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<td-paging-bar #pagingBarInput pageLinkCount="5" [firstLast]="false" [pageSize]="100" [total]="1345" (change)="changeInput($event)">
+  <p hide-xs>Go to:</p>
+  <mat-form-field [style.width.px]="30">
+    <input #goToInput 
+      matInput
+      type="number"
+      [min]="1"
+      [max]="pagingBarInput.maxPage"
+      [value]="pagingBarInput.page"
+      (blur)="goToInput.value = pagingBarInput.page"
```
Comment (christianmemije) [inline src/app/content/components/component-demos/paging/demos/paging-demo-input/paging-demo-input.component.html]: Let's use 
`(change)="pagingBarInput.navigateToPage(goToInput.value); goToInput.value = pagingBarInput.page"` which takes care or on blur and on enter

Comment (kriswinbush) [reply src/app/content/components/component-demos/paging/demos/paging-demo-input/paging-demo-input.component.html]: done

Comment (christianmemije) [reply src/app/content/components/component-demos/paging/demos/paging-demo-input/paging-demo-input.component.html]: You can now get rid of the keyup.enter

Comment (kriswinbush) [reply src/app/content/components/component-demos/paging/demos/paging-demo-input/paging-demo-input.component.html]: ..and done

**Pull request merged**